src/components/Finder/FinderTab.tsx

```
@@ -10,6 +10,109 @@ type Assumption = "bullish" | "neutral" | "bearish" | "volatile" | "pinning" | "
+
+  return (
+    <>
+      <button
+        onClick={(e) => { e.stopPropagation(); setOpen(true); }}
+        className="w-4 h-4 rounded-full bg-muted/50 hover:bg-muted text-muted-foreground/60 hover:text-foreground text-[9px] font-bold flex items-center justify-center shrink-0 transition-colors ml-1"
+        title={`About ${name}`}
+      >
+        i
+      </button>
+      {open && (
+        <div className="fixed inset-0 z-50 flex items-center justify-center bg-black/40" onClick={() => setOpen(false)}>
```

Biome (2.4.6)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)